CHANGELOG.md

```diff
@@ -1,3 +1,431 @@
+  worker_pool.node_type configuration options, upgrading to this release will
+  replace your controller and/or worker nodes with c3.small.x86 machines. To
+  avoid this, set these configuration options to the desired values.
+
+##### AKS
+
+- Update Kubernetes version to 1.16.10
+    ([#712](https://github.com/kinvolk/lokomotive/pull/712)).
+
+#### Component updates
+
+- openebs: update to 1.11.0 ([#673](https://github.com/kinvolk/lokomotive/pull/673)).
+
+- calico: update to
+  [v3.15.0](https://docs.projectcalico.org/release-notes/#v3150)
+  ([#652](https://github.com/kinvolk/lokomotive/pull/652)).
+
+#### UX
+
+- prometheus-operator: Organize prometheus related variables under a
+  `prometheus` block in the configuration
+  ([#710](https://github.com/kinvolk/lokomotive/pull/710)).
+
+- contour: Remove `ingress_hosts` from contour configuration
+    ([#635](https://github.com/kinvolk/lokomotive/pull/635)).
+
+#### Features
+
+- Add `enable_toolbox` variable to [rook-ceph
+component](https://github.com/kinvolk/lokomotive/blob/v0.3.0/docs/configuration-reference/components/rook-ceph)
+([#649](https://github.com/kinvolk/lokomotive/pull/649)). This allows Rook Ceph
+toolbox to manage and configure Ceph.
+
+- Use ingress to expose Prometheus and `external_labels` for federated clusters
+    ([#710](https://github.com/kinvolk/lokomotive/pull/710)).
+
+#### Docs
+
+- Add `Type` column to Attribute reference table in [configuration
+ references](https://github.com/kinvolk/lokomotive/blob/v0.3.0/docs/configuration-reference)
+ ([#651](https://github.com/kinvolk/lokomotive/pull/651)).
+
+- Update contour configuration reference for usage with AWS
+   ([#674](https://github.com/kinvolk/lokomotive/pull/674)).
+
+- Add documentation related to the usage of `clc_snippets` for Packet and AWS
+    ([#657](https://github.com/kinvolk/lokomotive/pull/657)).
+
+- Improve documentation on using remote
+    [backends](https://github.com/kinvolk/lokomotive/blob/v0.3.0/docs/configuration-reference/backend/s3.md)
+    ([#670](https://github.com/kinvolk/lokomotive/pull/670)).
+
+- How to guide for setting up monitoring on Lokomotive
+    ([#480](https://github.com/kinvolk/lokomotive/pull/480)).
+
+- Add `codespell` section in [development
+    documentation](https://github.com/kinvolk/lokomotive/blob/v0.3.0/docs/development/README.md)
+    ([#700](https://github.com/kinvolk/lokomotive/pull/700)).
+
+#### Bugfixes
+
+- Remove contour ingress workaround(due to an [upstream issue](https://github.com/projectcontour/contour/issues/403))
+  for ExternalDNS ([#635](https://github.com/kinvolk/lokomotive/pull/635)).
+
+#### Development
+
+- Must use consistent build tags for all platforms for CI
+    ([#600](https://github.com/kinvolk/lokomotive/pull/600))
+    & ([#676](https://github.com/kinvolk/lokomotive/pull/676)).
+
+- Disable marking releases as `pre-releases` ([#678](https://github.com/kinvolk/lokomotive/pull/678)).
```

Comment:
What? Why would a user care?

Reply:
my thoughts exactly ? Why a user would care ?

Reply:
According to https://docs.github.com/en/github/administering-a-repository/managing-releases-in-a-repository:
>To notify users that the release is not ready for production and may be unstable, select This is a pre-release.

Previously, all releases have been marked as `pre-release`, which "meant" that they were not ready for production use. Now we do that anymore, which may indicate, that they are actually ready.

However, our motivation is a change in GitHub UI, which if you only have pre-releases, won't show any releases on the main repository page, which is not desired.

I know it's probably a small detail, but in my opinion it can be interpreted as important change.

Alternatively, maybe we don't need to mention that at all, as it has been changed for all releases we have.

Reply:
If thats the case, then I'd lean on not including it in the release changelog. what do others think ?

Reply:
removed.